server/views/pages/index.njk

```diff
@@ -6,7 +6,7 @@
   }
 ] %}
 {% set pageTitle = applicationName + " - Home" %}
-{% set mainClasses = "govuk-body" %}
+{% set mainClasses = "govuk-body js-enabled" %}
```

Comment:
curious what this change was for? 🤔

Reply:
I was playing with the back to top button POC from style guide site - forgot to remove it 🙂